dataprofiler/profilers/profiler_options.py

```diff
@@ -982,6 +982,74 @@ def _validate_helper(self, variable_path: str = "CorrelationOptions") -> list[st
         return errors
 
 
+class HyperLogLogOptions(BooleanOption):
```

Comment:
agree -- let's make this a `UniqueCountOptions` with more granular toggles of `full` or `hll_row_hashing` options beneath that.